crates/meroctl/src/common.rs

```diff
@@ -59,6 +61,27 @@ where
         .map_err(|_| eyre!("Error with client request"))
 }
 
+pub fn load_config(path: &Utf8PathBuf) -> EyreResult<ConfigFile> {
+    if !ConfigFile::exists(&path) {
+        bail!("Config file does not exist")
+    };
+
+    let Ok(config) = ConfigFile::load(&path) else {
+        bail!("Failed to load config file")
+    };
+
+    Ok(config)
+}
+
+pub fn load_multiaddr(config: &ConfigFile) -> EyreResult<Multiaddr> {
+    let Some(multiaddr) = config.network.server.listen.first() else {
+        bail!("No address.")
+    };
+
+    Ok(multiaddr.clone())
+}
```

Comment:
I think we can skip this one, especially since the function name isn't very descriptive of what it aids, but if we chose to keep it, we can rename it and change the return type

```suggestion
pub fn load_multiaddr(config: &ConfigFile) -> EyreResult<&Multiaddr> {
    let Some(multiaddr) = config.network.server.listen.first() else {
        bail!("No address.")
    };

    Ok(multiaddr)
```

Reply:
yes, I will clean this up in the following PRs of method naming, for new lets leave it be until i go through rest of code. Changed to return refernce.